sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -100,6 +100,12 @@ object RapidsPluginUtils extends Logging {
       "More information is available at https://nvidia.github.io/spark-rapids/docs/FAQ.html#" +
       "automatic-translation-of-scala-udfs-to-apache-spark-operations" )
     }
+
+    logWarning("spark.rapids.sql.explain is set to the default value 'NOT_ON_GPU' to print " +
+        "operations not executed on GPU. Set to 'ALL' to print the complete query plan, " +
+        "display whether each operation is placed on the GPU. Set to 'NONE' to suppress the " +
+        "diagnostics about the query placement on the GPU.")
```

Comment:
I think we should move it inside the if branch `if (conf.isSqlEnabled && conf.isSqlExecuteOnGPU) {` on L85 so it prints only if GPU execution is enabled (default). 

And it should only be printed when the value for spark.rapids.sql.explain implies some output (this is an evolution from my prior review) to make it shorter. 

How about around L85:

```scala
if (conf.isSqlEnabled && conf.isSqlExecuteOnGPU) {
  logWarning("RAPIDS Accelerator is enabled, to disable GPU " +
    s"support set `${RapidsConf.SQL_ENABLED}` to false.")

  if (conf.explain != 'NONE') {
    logWarning("spark.rapids.sql.explain is set to ${conf.explain}. Set to 'NONE' to suppress the " +
        "diagnostics logging about the query placement on the GPU.")
  }
} else ...
```

Reply:
I have updated the code here. Thanks.